tasks/post-deploy/08-gravityforms.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "Set GF settings..."
+wp option add rg_gforms_key "${GF_LICENSE}" || wp option update rg_gforms_key "${GF_LICENSE}"
```

Comment:
Should be sufficient to only have `wp option update ...`. It also creates the option if it doesn't exist.